api/rs/slint/Cargo.toml

```diff
@@ -56,6 +56,9 @@ serde = ["i-slint-core/serde"]
 ## This feature enables the software renderer to pick up fonts from the operating system for text rendering.
 software-renderer-systemfonts = ["renderer-software", "i-slint-core/software-renderer-systemfonts"]
 
+## Internal feature: Enable shared fontique font handling
+shared-fontique = ["i-slint-core/shared-fontique"]
```

Comment:
If this feature is meant to be internal , it should not be documented and have a name that shows it.
Example: https://github.com/slint-ui/slint/blob/55a45a6c339223a19bcd79fc191e79193f495a79/api/rs/build/Cargo.toml#L28C1-L28C14

Maybe something like `experimental-register-font`

But actually, i'd rather not have experimental API like this enabled from the slint crate, but directly on the i-slint-core crate, and have app that want to use internal API to use the i-slint-core directly.